tests/phpunit/tests/test-wpml-gutenberg-string-registration.php

```diff
@@ -143,6 +143,64 @@ public function it_registers_strings_and_set_location() {
 		$subject->register_strings( $post, $package );
 	}
 
+	/**
+	 * @test
+	 * @group wpmltm-3081
+	 */
+	public function it_registers_strings_and_set_wrap_tag() {
+		$post               = \Mockery::mock( 'WP_Post' );
+		$post->post_content = '<!-- wp:something -->post content<!-- /wp:something -->';
+
+		$package = array(
+			'kind' => WPML_Gutenberg_Integration::PACKAGE_ID,
+		);
+
+		$blocks = array();
+
+		$blocks['block 1']                 = \Mockery::mock( 'WP_Block_Parser_Block' );
+		$blocks['block 1']->blockName      = 'core/heading';
+		$blocks['block 1']->innerHTML      = 'some block content 1';
+		$blocks['block 1']->attrs['level'] = 1;
+		$string_name_1                     = md5( $blocks['block 1']->blockName . $blocks['block 1']->innerHTML );
+		$string_id_1                       = 123;
+
+		$blocks['block 2']                 = \Mockery::mock( 'WP_Block_Parser_Block' );
+		$blocks['block 2']->blockName      = 'core/heading';
+		$blocks['block 2']->innerHTML      = 'some block content 2';
+		$blocks['block 2']->attrs['level'] = 2;
+		$string_name_2                     = md5( $blocks['block 2']->blockName . $blocks['block 2']->innerHTML );
+		$string_id_2                       = 456;
```

Comment:
@kagg-design I would reduce some duplication here.

1. Create a private helper function:  
   ```php
	/**
	 * @return \WP_Block_Parser_Block $block|\Mockery\MockInterface
	 */
	private function get_block( $name, $html ) {
		/** @var \WP_Block_Parser_Block $block|\Mockery\MockInterface */
		$block            = \Mockery::mock( 'WP_Block_Parser_Block' );
		$block->blockName = $name;
		$block->innerHTML = $html;


		return $block;
	}
   ```
2. Use it:  
   ```php
		$blocks = array(
			'block 1' => $this->get_block( 'some name 1', 'some block content 1' ),
			'block 2' => $this->get_block( 'some name 2', 'some block content 2' ),
		);

		$string_name_1 = md5( $blocks['block 1']->blockName . $blocks['block 1']->innerHTML );
		$string_id_1   = 123;

		$string_name_2 = md5( $blocks['block 2']->blockName . $blocks['block 2']->innerHTML );
		$string_id_2   = 456;
   ```

Reply:
@andreasciamanna thank you for pointing on it. Done in my test and all already existing tests. Commit 18847698533c5374042f319e53aac068678b1aeb